SentryCapacitor.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.author = package['author']
   s.source = { :git => package['repository']['url'], :tag => s.version.to_s }
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
-  s.ios.deployment_target  = '12.0'
+  s.ios.deployment_target  = '13.0'
```

Comment:
You can check using the following code

```ruby
capacitorPackage =  JSON.parse(File.read(File.join(__dir__, '../../@capacitor/core/package.json')))
capacitorVersion = capacitorPackage['version']
if capacitorVersion.start_with?("2.") or capacitorVersion.start_with?("3.")
  miniOSVersion = '12.0'
else 
  miniOSVersion = '13.0' # Required for Capacitor 4 and newer.
end```

```ruby
  s.ios.deployment_target  = miniOSVersion

```

I believe with that we can keep the support for old devices as long as people use the old Capacitor.